src/Services/DataService.test.ts

```diff
@@ -604,13 +610,58 @@ describe("DataService", () => {
       fakeService.fetchRecordEpic(ActionsObservable.of(fetchRecordAction), store)
         .subscribe(noop, noop,
           () => {
-            assert.isTrue(mockAdapter.fetchItem.called);
+            expect(mockAdapter.fetchItem.called).to.be.true;
+          });
+    });
+
+    it("fetchItem should call normalize after serialize", () => {
+      const nonCachedItemId = 51;
+      const deserializedObject = { name: "Zella puppy" };
+      const fetchRecordAction = fakeService.actions.fetchRecord({ id: nonCachedItemId }, null);
+      const normalizedStub = stub(fakeService.mapper, "normalize");
+      
+      stub(fakeService.serializer, "deserialize").returns (deserializedObject);
+
+      fakeService.fetchRecordEpic(ActionsObservable.of(fetchRecordAction), store)
+        .subscribe(noop, noop,
+          () => {
+            expect(normalizedStub.firstCall.args[0]).to.equal(deserializedObject);
+          });
+    });
+
+    it("fetchItem should call transform before serialize", () => {
```

Comment:
fetchRecord shouldn't be calling serialize or transform.  If you set a breakpoint on the expect, I suspect that it won't hit that statement due to an error.